### DISCUSSION TITLE
Two questions for the scoop/windows version

### DISCUSSION THREAD

**Discussion by Sourpunch14** [gorilla-devs/ferium#245] [Q&A] [answered]
Is there no UI? I've just been running commands through the powershell terminal. And also where is the user folder to manually install mods? I've tried creating one in both .minecraft and .minecraft\mods but when running ferium upgrade it just gives an error "Your currently selected profile is empty! Run `ferium help` to see how to add mods"

**Comment by theRookieCoder** [comment, marked as answer]
> Is there no UI?

yes ferium is a _CLI_ minecraft mod manager.

> where is the user folder

The user folder is in the output directory, so `.minecraft/mods/user` should be correct. The error you get is completely unrelated.

> it just gives an error `Your currently selected profile is empty! Run ferium help to see how to add mods`

You have to tell ferium what mods to download. Unfortunately ferium can't create a list of mods from your mods folder yet (see #57)

**Reply by Sourpunch14** [reply]
Oh, I'm dumb and missed the CLI part. For the other two things you answered, am I misunderstanding how this works? "External mods
If you want to use files that are not downloadable by ferium, place them in the user folder in the output directory. Files here will be copied to the output directory when upgrading" This is under your "Adding mods" section

 New question, is there a way to install multiple mods at once? I want to use this to simplify sharing mods with my partner and I don't mind if I have to, say, manually type out a command like "ferium add sodium, phosphor, lithium"?

**Reply by theRookieCoder** [reply]
> I misunderstanding how this works

Could you elaborate? I'm not sure what you're confused about.

> install multiple mods at once

I think you meant _adding_ multiple mods at once? That's a planned feature (see #175)

**Reply by Sourpunch14** [reply]
> "External mods If you want to use files that are not downloadable by ferium, place them in the user folder in the output directory. Files here will be copied to the output directory when upgrading"

The way I understand this is that I add .jar mod files to .minecraft\mods\user and then type ferium upgrade and it then copies the files to the mods folder, is that correct? If so, then the issue I'm having is that doing so gives me the error of the profile being empty.

> I think you meant adding multiple mods at once? That's a planned feature (see https://github.com/gorilla-devs/ferium/issues/175)

Yeah, I meant adding. Thank you!

**Comment by theRookieCoder** [comment]
> The way I understand this is that I add .jar mod files to .minecraft\mods\user and then type ferium upgrade and it then copies the files to the mods folder, is that correct?

Well yes that's how it works, however that is not the main function of the program. Ferium is supposed to be provided with a list of mods (using `ferium add`), then these mods are downloaded by the program into `.minecraft/mods`. You have to add mods for ferium to download first, only then will the `ferium upgrade` command work.

**Reply by Sourpunch14** [reply]
Ahh. I just wanted to copy mods over from when I had separate folders of mods. I see now
